shopify_function/src/scalars.rs

```diff
@@ -5,3 +5,4 @@ pub type ID = String;
 pub type Decimal = String;
 pub type Void = ();
 pub type URL = String;
+pub type ShortID = String;
```

Comment:
This is the question I have top of mind:

```suggestion
pub type ShortID = i64;
```

While ShortIDs are documented to be used as opaque token, we are introducing them for performance reasons. 

I also recognize that they will be used as database IDs in Functions.

Representing them as `i64` would provide better performance both in terms of the transport layer and execution logic, correct?

Reply:
> (maybe it was just on [GQL side?](https://github.com/Shopify/shopify/pull/444657/files))

That's where my "I don't have the GraphQL knowledge to review this specific piece of code." comes in.

I know there was certain limits with GraphQL at Shopify scale. I though it was int overflow but I could be wrong, it is possible this was for Floats serialization and unrelated.

Reply:
I asked @surma about this: Javy only really supports f64, which would give us a max of a 53 bit int, or we'd have to use BigInts which is likely to erode the performance impact of not using a string and increase complexity, so let's stick with strings for now!